src/Profile/Profile.js

```diff
@@ -70,100 +92,138 @@ function ResidentCard({resident, exitProfileView}) {
-          <Typography align='left' variant="h6" color="textSecondary">
-            Other Impairments:{`${resident.other_impairments}`}
+          <Typography variant="h6" color="textSecondary" gutterBottom>
+            Pets:
           </Typography>
+          <Typography paragraph>
+            {`${resident.pets}`}
+          </Typography>      
+          <Typography variant="h6" color="textSecondary" gutterBottom>
+            Living Preferences:
+          </Typography>      
+          <Typography paragraph>
+            {`${resident.living_preferences}`}
+          </Typography>     
+          <Typography variant="h6" color="textSecondary" gutterBottom>
+            Additional Notes:
+          </Typography>      
+          <Typography paragraph>
+            {`${resident.additional_notes}`}
+          </Typography>    
         </CardContent>
-      </Collapse>
+            
+          <CardActions disableSpacing>
+            {applied && 
+              <>
+                <img className={classes.heartImg} src={turingHealthLogo} alt="application-in-process" />
+                <Button variant="outlined" color="secondary" aria-label="application-pending">
+                    Application Pending
+                </Button>
+              </>
+            }
+            {!applied && 
+              <div>
+                <IconButton aria-label="no-application">
+                  <FavoriteIcon aria-label="no-application"/>
+                </IconButton>
+                <Button onClick={handleClickOpenDialogue} variant="outlined" color="secondary" aria-label="apply for housing">
+                    Apply for Housing
+                </Button>
+                <Dialog
+                  open={open}
+                  onClose={handleCloseDialogue}
+                  aria-labelledby="alert-dialog-apply-for-housing"
+                  aria-describedby="alert-dialog-agree-or-disagree"
+                >
+                  <DialogTitle id="alert-dialog-apply">Submit Application?</DialogTitle>
+                  <DialogContent>
+                  <DialogContentText id="alert-dialog-description">
+                    Choose "Agree" if you would like to submit your application to house with {resident.name} for the semester of {resident.semester}. You will be notified via email if you are selected for this residency position pending approval.  
+                  </DialogContentText>
+                  </DialogContent>
+                  <DialogActions>
+                    <Button onClick={handleCloseDialogue} color="secondary">
+                      Disagree
+                    </Button>
+                    <Button 
+                      onClick={handleApplyChange} 
+                      color="secondary" 
+                      autoFocus>
+                      Agree
+                    </Button>
+                  </DialogActions>
+                </Dialog>
+              </div>
+            }
+            <IconButton
```

Comment:
I feel like the medical icon button is way more representative!

Reply:
It looks so good! I concur!